core/src/main/java/com/axellience/vuegwt/core/client/tools/VueGWTTools.java

```diff
@@ -165,6 +166,7 @@ public static String templateExpressionToString(char[] expressionValue) {
         Any val = asAny(map.get(key));
         if (isTripleEqual(val, null) ||
             isTripleEqual(val, asAny(1)) ||
+            isTripleEqual(val, asAny(1L)) ||
```

Comment:
I'm not 100% confident in this... but it worked in vue-gwt-demo! Here's my concern - if longs are {high:0 low:1}, then `===` won't compare them properly. However, since you only assign these little constants, I think it works since closure-library caches certain small long values, which makes this work.

Open to feedback on how to better bulletproof this, how to write tests to confirm that it works long-term.

Reply:
Maybe using the same variable would make J2CL us the same reference for the long object and make it "safer"?
Like `LONG_VALUE = 1L` then using `LONG_VALUE` as the marker and `isTripleEqual(val, asAny(LONG_VALUE))` to check.
I will see to write a test for it though once it's merge in develop.